custom/intrahealth/tests/reports/test_dashboard_2.py

```diff
@@ -240,16 +240,16 @@ def test_recovery_rate_by_pps_report_country_level(self):
                  {'html': 'pas de donn\xe9es'}, {'html': 'pas de donn\xe9es'}, {'html': 'pas de donn\xe9es'},
                  {'html': '0.00%'}, {'html': '0.00%'}],
                 [{'html': 'Fatick'}, {'html': 'pas de donn\xe9es'}, {'html': '90.47%'}, {'html': '2.75%'},
-                 {'html': '0.00%'}, {'html': 'pas de donn\xe9es'}, {'html': '0.00%'}, {'html': '23.82%'}],
+                 {'html': '0.00%'}, {'html': 'pas de donn\xe9es'}, {'html': '0.00%'}, {'html': '29.88%'}],
                 [{'html': 'Region 1'}, {'html': 'pas de donn\xe9es'}, {'html': 'pas de donn\xe9es'},
                  {'html': 'pas de donn\xe9es'}, {'html': 'pas de donn\xe9es'}, {'html': 'pas de donn\xe9es'},
-                 {'html': '46.15%'}, {'html': '46.15%'}],
+                 {'html': '92.31%'}, {'html': '92.31%'}],
                 [{'html': 'Region Test'}, {'html': 'pas de donn\xe9es'}, {'html': 'pas de donn\xe9es'},
-                 {'html': 'pas de donn\xe9es'}, {'html': 'pas de donn\xe9es'}, {'html': '27.50%'},
-                 {'html': 'pas de donn\xe9es'}, {'html': '27.50%'}],
-                [{'html': 'Saint-Louis'}, {'html': '63.85%'}, {'html': '87.68%'}, {'html': '0.00%'},
+                 {'html': 'pas de donn\xe9es'}, {'html': 'pas de donn\xe9es'}, {'html': '191.90%'},
+                 {'html': 'pas de donn\xe9es'}, {'html': '191.90%'}],
```

Comment:
numbers over 100% seem off. Is that expected here?

Reply:
I have to look into that once again, as I have been making the case based on this record (Region Test) and 103% was expected (bills were slightly overpaid).